### ISSUE TITLE
new version 0.6.0 cannot support StaggeredGridView.countBuilder function

### ISSUE THREAD

**Issue by chhua531001** [letsar/flutter_staggered_grid_view#223]
Sir, why the new version 0.6.0 StaggeredGridView.countBuilder function, i hope the next version, can support this function, thank you.

**Comment by letsar** [comment]
No it won't (see https://github.com/letsar/flutter_staggered_grid_view/discussions/210). But you may want to use MasonryGridView instead.

**Comment by satrio1256** [comment]
@letsar maybe you can add some docs in the readme on how to migrate to the new method? I guess some of the users that use your library is previously using `StaggeredGridView.countBuilder`

**Comment by huhx** [comment]
> @letsar maybe you can add some docs in the readme on how to migrate to the new method? I guess some of the users that use your library is previously using `StaggeredGridView.countBuilder`

Agree, Especially, If we use staggeredTileBuilder in StaggeredGridView.countBuilder before. +1

**Comment by JuYiYang** [comment]
> staggeredTileBuilder

Hello, I want to ask the new version unbind controller, how do I know to scroll to the bottom? What I'm doing now is I'm putting it in the listview but you seem to have a mechanism where you don't render until you're in the viewable area, which leads to being at the bottom all the time

**Comment by JuYiYang** [comment]
> No it won't (see #210). But you may want to use MasonryGridView instead.

Hello, I want to ask the new version to cancel the controller property, how do I know to scroll to the bottom? What I'm doing now is I'm putting it in the listview but it looks like you've made a mechanism to not render pixels before you've reached the viewable area, which causes the listview to listen to Pixels all the time at the bottom
